timescaledb/quick-start/golang.md

```diff
@@ -1,293 +1,266 @@
+measurements contain the time, sensor_id, temperature reading, and CPU
+percentage of the sensors.
+
+<procedure>
+
+<Collapsible heading="Creating a relational table" headingLevel={3}>
+
+1.  Compose a string that contains the SQL statement to create a relational
+    table. This example creates a table called `sensors`, with columns for ID,
+    type, and location:
+
+    ```go
+    queryCreateTable := `CREATE TABLE sensors (id SERIAL PRIMARY KEY, type VARCHAR(50), location VARCHAR(50));`
+    ```
+
+1.  Execute the `CREATE TABLE` statement with the `Exec()` function on the
+    `dbpool` object, using the arguments of the current context and the
+    statement string you created:
+
+    ```go
+    package main
+
+    import (
+       "context"
+       "fmt"
+       "os"
+
+       "github.com/jackc/pgx/v5"
+       "github.com/jackc/pgx/v5/pgxpool"
+    )
+
+    func main() {
+
+       ctx := context.Background()
+       connStr := "yourConnectionStringHere"
+       dbpool, err := pgxpool.New(ctx, connStr)
+       if err != nil {
+           fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
+           os.Exit(1)
+       }
+       defer dbpool.Close()
 
-   ctx := context.Background()
-   connStr := "yourConnectionStringHere"
-   dbpool, err := pgxpool.Connect(ctx, connStr)
-   if err != nil {
-       fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
-       os.Exit(1)
-   }
-   defer dbpool.Close()
+       /********************************************/
+       /* Create relational table                      */
+       /********************************************/
 
-   /********************************************/
-   /* Create relational table                      */
-   /********************************************/
+       //Create relational table called sensors
+       queryCreateTable := `CREATE TABLE sensors (id SERIAL PRIMARY KEY, type VARCHAR(50), location VARCHAR(50));`
+       _, err = dbpool.Exec(ctx, queryCreateTable)
+       if err != nil {
+           fmt.Fprintf(os.Stderr, "Unable to create SENSORS table: %v\n", err)
+           os.Exit(1)
+       }
+       fmt.Println("Successfully created relational table SENSORS")
+    }
+       ```
 
-   //Create relational table called sensors
-   queryCreateTable := `CREATE TABLE sensors (id SERIAL PRIMARY KEY, type VARCHAR(50), location VARCHAR(50));`
-   _, err = dbpool.Exec(ctx, queryCreateTable)
-   if err != nil {
-       fmt.Fprintf(os.Stderr, "Unable to create SENSORS table: %v\n", err)
-       os.Exit(1)
-   }
-   fmt.Println("Successfully created relational table SENSORS")
+</Collapsible>
 
-}
-```
+</procedure>
 
 ## Generate a hypertable
 
-In TimescaleDB, the primary point of interaction with your data is
-a hypertable, the abstraction of a single continuous table across all
-space and time intervals, such that one can query it via standard SQL.
+Virtually all user interactions with TimescaleDB are with hypertables. Creating
```

Comment:
Do we like this statement? I don't love it, I don't know how true it is, because you can have tons of relational tables for metadata as well

Reply:
Yeah, this is old text. It made it into the Python one too. I wonder if we can just delete it.

Reply:
Totally passed me by on that way. Would be up for deleting it